src/components/ceremony/ceremony.repository.ts

```diff
@@ -32,15 +32,20 @@ export class CeremonyRepository extends DtoRepository<
       actualDate: input.actualDate,
       canDelete: true,
     };
-    return await this.db
+    const result = await this.db
       .query()
       .apply(await createNode(Ceremony, { initialProps }))
-      .return<{ id: ID }>('node.id as id')
+      .return<{ node: UnsecuredDto<Ceremony> }>('node')
       .first();
+
+    if (!result) {
+      throw new ServerException('failed to create a ceremony');
+    }
+    return result;
   }
 
   async update(
-    existing: Ceremony,
+    existing: DbTypeOf<Ceremony>,
```

Comment:
Follow this pattern instead for updates.
https://github.com/SeedCompany/cord-api-v3/blob/develop/src/components/funding-account/funding-account.service.ts#L103-L104
https://github.com/SeedCompany/cord-api-v3/blob/develop/src/components/funding-account/funding-account.repository.ts#L38-L38